### PR TITLE
Delete service and wait for NEG deletion in ASM e2e tests

### PR DESCRIPTION
**Pull request by panslava**

### CODE REVIEW

File: pkg/e2e/fixtures.go [hunk 1]
```diff
@@ -452,6 +452,11 @@ func CreatePorterService(s *Sandbox, name string) error {
 	return err
 }
 
+// DeletePorterService deletes the K8s porter service
+func DeletePorterService(s *Sandbox, svcName string) error {
```
Comment (freehan) [inline pkg/e2e/fixtures.go:456]: Is this used?

**Review by freehan — COMMENTED** [review]
ping me when fixed

File: pkg/e2e/fixtures.go (hunk 1 above)
Comment (panslava) [reply pkg/e2e/fixtures.go]: No, forgot to remove... Thanks

**Pull request merged**